test/model-method.js

```diff
@@ -101,3 +101,112 @@ describe('ModelMethod', function() {
       });
   });
 });
+
+describe('ModelMethod - Loopback 3.0', function() {
+  var userModel;
+
+  beforeEach(function(done) {
+    givenLBXWorkspace({'loopback':'^3.0.0'}, done);
+  });
+  
+  beforeEach(function(done) {
+    ModelDefinition.create(
+      {
+        name: 'user',
+        facetName: 'server'
+      },
+      function(err, result) {
+        if (err) return done(err);
+        userModel = result;
+        done();
+      });
+  });
+ 
+  it('add static method without isStatic flag to model definition', function(done) {
+    ModelMethod.create(
+      {
+        modelId: userModel.id,
+        name: 'testMethod',
+        isStatic: true
+      },
+      function(err) {
+        if (err) return done(err);
+        userModel.methods(function(err, list) {
+          if (err) return done(err);
+          expect(list).to.have.length(1);
+          expect(list[0]).to.have.property('name', 'testMethod');
+          expect(list[0]).to.not.have.property('isStatic');
+
+          var cfg = new ConfigFile({ path: 'server/models/user.json' });
+          cfg.load(function(err) {
+            if (err) return done(err);
+            var methods = cfg.data.methods;
+            expect(methods).to.be.an('object');
+            expect(methods).to.have.property('testMethod');
+            expect(methods).to.not.have.property('prototype.testMethod');
+            expect(methods).to.not.have.property('isStatic');
```

Comment:
You are checking that "methods" object don't contain `isStatic` flag, I think it's more important to test that method definition does not contain this flag. Thoughts?

```js
expect(methods.testMethod).to.not.have.property('isStatic');
```